security/certManager/config.go

```diff
@@ -3,30 +3,17 @@ package certManager
 import (
 	"crypto/tls"
 	"fmt"
-
-	"github.com/plgd-dev/kit/security/certManager/acme"
-	"github.com/plgd-dev/kit/security/certManager/acme/ocf"
-	"github.com/plgd-dev/kit/security/certManager/file"
 )
 
-// AcmeType define acme type certificate manager
-const AcmeType = "acme"
-
-// FileType define static file type certificate manager
-const FileType = "file"
-
-// Config provides configuration of a file/acme based Certificate manager
+// Config provides configuration of a file based Certificate manager
 type Config struct {
-	Type string      `envconfig:"TYPE" default:"acme"`
-	Acme acme.Config `envconfig:"ACME"`
-	File file.Config `envconfig:"FILE"`
-}
-
-// OcfConfig provides configuration of a file/acme based Certificate manager
-type OcfConfig struct {
-	Type string      `envconfig:"TYPE" default:"acme"`
-	Acme ocf.Config  `envconfig:"ACME"`
-	File file.Config `envconfig:"FILE"`
+	Enabled                   	bool 	`yaml:"enabled" json:"enabled" default:"true"`
+	CAFile                    	string 	`yaml:"ca-file" json:"ca-file" description:"file path to the root certificate in PEM format"`
+	KeyFile                   	string 	`yaml:"key-file" json:"key-file" description:"file name of private key in PEM format"`
+	DirPath                   	string 	`yaml:"dir-path" json:"dir-path" description:"dir path where cert/key pair are saved"`
```

Comment:
we want to remove `dir-path` -> file path will be stored in `ca-file, key-file, cert-file` . So we need to watch the directories which are mentioned in the file paths.